app/src/test/java/com/keylesspalace/tusky/util/LocaleUtilsTest.kt

```diff
@@ -0,0 +1,82 @@
+package com.keylesspalace.tusky.util
+
+import androidx.appcompat.app.AppCompatDelegate
+import androidx.core.os.LocaleListCompat
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import com.keylesspalace.tusky.db.AccountEntity
+import org.junit.Assert
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.mockito.Mockito
+import org.robolectric.annotation.Config
+
+@Config(sdk = [28])
+@RunWith(AndroidJUnit4::class)
+class LocaleUtilsTest {
+    @Test
+    fun initialLanguagesContainReplySelectedAppAndSystem() {
+        val expectedLanguages = arrayOf<String?>("yi", "tok", "da", "fr", "sv", "kab")
+        val languages = getMockedInitialLanguages(expectedLanguages)
```

Comment:
Shouldn't a test simply call the normal method with some test data?

(One wonders what is tested about a mocked list.)

Reply:
This function is a wrapper that mocks the response for querying the configured app and system languages, so that:
- the tests don't accidentally rely on whatever values robolectric returns
- we can verify the behavior of the code when it returns certain types of values